packages/mobile-top-nav/src/media-slider.js

```diff
@@ -1,14 +1,73 @@
-import { LitElement, html, css } from 'lit-element';
+import { LitElement, html } from 'lit-element';
+import mediaSliderCss from './css/media-slider';
+import menus from './data/menus';
+import './wayback-search';
+import './more-slider';
 
 class MediaSlider extends LitElement {
+  constructor() {
+    super();
+    const defaultLinks = { iconLinks: [], featuredLinks: [], links: [] };
+
+    this.mediaSliderOpen = false;
+    this.mediaSliderAnimate = false;
+    this.selectedMenuOption = 'texts';
+
+    // Begin properties not monitored by LitElement
+    this.links = menus[this.selectedMenuOption] || defaultLinks;
+    this.templates = {
+      web: () => html`<wayback-search></wayback-search>`,
+      more: () => html`<more-slider></more-slider>`,
+    };
+  }
+
   static get properties() {
     return {
       mediaSliderOpen: { type: Boolean },
       mediaSliderAnimate: { type: Boolean },
-      links: { type: Array },
+      selectedMenuOption: { type: String },
     };
   }
 
+  shouldUpdate() {
+    if (menus[this.selectedMenuOption]) {
+      this.links = menus[this.selectedMenuOption];
+    }
+    return true;
+  }
+
+  renderIconLinks() {
```

Comment:
For getters like this you can use this syntax:
```
get iconLinks() {
  return html`...`;
}
```
then in your `render()` method:
```
return html`
  ${this.iconLinks}
`;
```